files/es/web/html/element/object/index.md

```diff
@@ -71,8 +71,8 @@ La fuente original de este ejemplo interactivo está almacenada en un repositori
     <tr>
       <th scope="row">Permitted ARIA roles</th>
       <td>
-        {{ARIARole("application")}}, {{ARIARole("document")}},
-        {{ARIARole("image")}}
+        <a href="/es/docs/Web/Accessibility/ARIA/Roles/application_role"><code>application</code></a>, <a href="/es/docs/Web/Accessibility/ARIA/Roles/document_role"><code>document</code></a>,
+        <a href="/es/docs/Web/Accessibility/ARIA/Roles/image_role"><code>image</code></a>
```

Comment:
It maybe should be `img` as the file above, suggesting the change to in `en-US` in https://github.com/mdn/content/pull/23869
```suggestion
        <a href="/es/docs/Web/Accessibility/ARIA/Roles/img_role"><code>img</code></a>
```